### PR TITLE
Update inline link story and bump packge info for Caption

### LINKED ISSUE #720: Caption story for inline link tweaks
**Is your feature request related to a problem? Please describe.**
We should update the "caption with inline link" so that the link is between 2 items of text to give a better example of how the inline link styling affects both the line of text both above and below the link. 

**Describe the solution you'd like**
- Add a second line of caption text (potentially a duplicate) 
- Tidy up the grammar of the stories EG: add a space between the inline link component and the caption text

**Describe alternatives you've considered**
We could split the text inputs before and after the inline link to give better control but that would include changing all the translations which might be a future optimisation.

**Testing notes**
[Tester to complete]

Dev insight: Regression test storybook caption inline link story

**Additional context**
Raised while working on https://github.com/bbc/psammead/pull/707

### PR DESCRIPTION
**Pull request by pjlee11**
Resolves #720

**Overall change:** _Updates the caption story to give a better example of usage with the inline link. Also bumps @bbc/psammead-inline-link to v1.1.0_

**Code changes:**

- _Adds a second line of text so the inline link has both text content before and after it_
- _Updates package.json, package-lock.json and changelog_

---

- [x] I have assigned myself to this PR and the corresponding issues
- [x] Tests added for new features
- [ ] Test engineer approval

### CODE REVIEW

File: packages/components/psammead-caption/CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
 <!-- prettier-ignore -->
 | Version | Description |
 |---------|-------------|
+| 1.1.6 | [PR#xxx](https://github.com/bbc/psammead/pull/xxx) Use `@bbc/psammead-inline-link@1.1.0` and update 'with inline link' story |
```
Comment (pjlee11) [inline packages/components/psammead-caption/CHANGELOG.md]:
```suggestion
| 1.1.6 | [PR#745](https://github.com/bbc/psammead/pull/745) Use `@bbc/psammead-inline-link@1.1.0` and update 'with inline link' story |
```

File: packages/components/psammead-caption/package.json [hunk 2]
```diff
@@ -21,7 +21,7 @@
     "@bbc/psammead-styles": "^0.3.2"
   },
   "devDependencies": {
-    "@bbc/psammead-inline-link": "^0.3.4",
+    "@bbc/psammead-inline-link": "^1.0.0",
```
Comment (pjlee11) [inline packages/components/psammead-caption/package.json]:
```suggestion
    "@bbc/psammead-inline-link": "^1.1.0",
```

Bump inline-link after https://github.com/bbc/psammead/pull/707 is merged and published and `npm install`

**Comment by pjlee11** [comment]
Blocked on https://github.com/bbc/psammead/pull/707

**Review by j-pendlebury — APPROVED** [review]

File: packages/components/psammead-caption/src/index.stories.jsx [hunk 3]
```diff
@@ -41,8 +41,9 @@ storiesOf('Components|Caption', module)
       ],
       ([linkText, captionText], script) => (
         <Caption script={script}>
-          {captionText}
-          <InlineLink href="https://www.bbc.com"> {linkText}</InlineLink>.
+          {`${captionText} `}
+          <InlineLink href="https://www.bbc.com">{linkText}</InlineLink>
+          {` ${captionText} `}
```
Comment (oluoluoxenfree) [inline packages/components/psammead-caption/src/index.stories.jsx:46]: Why is caption text here twice?

Comment (pjlee11) [reply packages/components/psammead-caption/src/index.stories.jsx:46]: As per the `Code changes` in the description of the PR:
>Adds a second line of text so the inline link has both text content before and after it

**Review by oluoluoxenfree — APPROVED** [review]

**Comment by pjlee11** [comment]
image

**Comment by jamesbrumpton** [comment]
LGTM

**Pull request merged**